src/style.css

```
@@ -1,3 +1,75 @@
+html {
+  --accent-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo_img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav_list {
+  display: flex;
+  align-content: center;
   margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  position: relative;
```

do we need positioning for each element?
would it be better to move this to our modifier?
```suggestion
  position: relative;
```